lua/null-ls/builtins/completion/vsnip.lua

```diff
@@ -11,14 +11,27 @@ return h.make_builtin({
         fn = function(params, done)
             local items = {}
             local snips = vim.fn["vsnip#get_complete_items"](params.bufnr)
-            local targets = vim.tbl_filter(function(item)
-                return string.match(item.word, "^" .. params.word_to_complete)
-            end, snips)
-            for _, item in ipairs(targets) do
+            for _, item in ipairs(snips) do
+                local user_data = vim.fn.json_decode(item.user_data)
```

Comment:
```suggestion
                local user_data = vim.json.decode(item.user_data)
```

#389 changes all uses of `vim.fn.json_decode` to `vim.json.decode`, which is faster and available on 0.6.0, so we might as well use it here, too.